.env.site-configs.tpl

```diff
@@ -0,0 +1,3 @@
+PRIVATE_SITE_CONFIGS='{{ site://configs/private/local }}'
+PUBLIC_SITE_CONFIGS='{{ site://configs/public/local }}'
+NEXT_PUBLIC_SITE_CONFIGS=$PUBLIC_SITE_CONFIGS
```

Comment:
Shouldn't everything that's related to Demo's site configs be in `demo/`?

Reply:
yes!

also docker-compose should be in demo/

Reply:
but that should be a different PR